scripts/install.js

```diff
@@ -56,6 +57,8 @@ function download(url, dest, cb) {
     }
   };
 
+  console.log('Start downloading binary at', url);
```

Comment:
Was the `console.log` intentional? Not sure if the `log.newGroup` makes sense here